deploy/dashboards/ipfs.json

```diff
@@ -595,104 +603,6 @@
               "type": "expression"
             }
           ]
-        },
```

Comment:
I think the intention is to have all agent versions compared to just go-ipfs agents in the previous graph. I might be wrong to delete these, but they seemed to me like they were selecting nearly the same thing. I kept "A", and deleted "B" and "C".